### PR TITLE
check quick start in CI

### PR DESCRIPTION
**Pull request by annashamray**
Fixes https://github.com/maykinmedia/objects-api/issues/91

### CODE REVIEW

File: .github/workflows/quick_start.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+on: [push]
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run quick_start.sh
+        run: ./bin/quick_start.sh
```
Comment (joeribekker) [inline .github/workflows/quick_start.yml:11]: Do we need checkout? We're downloading the docker-compose file?
and the quickstart script cannot be done with run-statements?

Comment (annashamray) [reply .github/workflows/quick_start.yml]: the github action is changed to use only run statements

**Review by joeribekker — CHANGES_REQUESTED** [review]
Please add the step to retrieve the index page, like step 2 in the quickstart, and that it returns an HTTP 200.

**Comment by annashamray** [comment]
@joeribekker index page request is added to the quick check

**Review by joeribekker — APPROVED** [review]

**Pull request merged**